src/Microsoft.Extensions.Logging.Abstractions/Internal/TypeNameHelper.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Microsoft.Extensions.Logging.Abstractions.Internal
+{
+    public class TypeNameHelper
+    {
+        private static readonly Dictionary<Type, string> _builtInTypeNames = new Dictionary<Type, string>
+            {
+            { typeof(bool), "bool" },
+            { typeof(byte), "byte" },
+            { typeof(char), "char" },
+            { typeof(decimal), "decimal" },
+            { typeof(double), "double" },
+            { typeof(float), "float" },
+            { typeof(int), "int" },
+            { typeof(long), "long" },
+            { typeof(object), "object" },
+            { typeof(sbyte), "sbyte" },
+            { typeof(short), "short" },
+            { typeof(string), "string" },
+            { typeof(uint), "uint" },
+            { typeof(ulong), "ulong" },
+            { typeof(ushort), "ushort" }
+            };
+
+        public static string GetTypeDisplayName(object item, bool fullName = true)
```

Comment:
:+1: this is not being used...we are always using full name in logging, so will remove it..